bot.js

```diff
@@ -122,323 +124,322 @@ var Helps = [
-            gold:0,
-            diamond:0,
+        inv: {
+            stone: 0,
+            coal: 0,
+            iron: 0,
+            gold: 0,
+            diamond: 0,
         },
         pick: 0,
         level: 1,
-        mp: {x:4,y:4},
+        mp: {
+            x: 4,
+            y: 4
+        },
         xp: 0,
         name: message.author.username.toString(),
     });
 };
-var addPickaxeRoles = function(message, I){
-    if(message.channel.type !== 'dm' && (bot.guilds.get(message.guild.id).id).toString() === "550036987772403714"){
-      if(I.pick >= 0){
-          var role = message.guild.roles.find(role => role.name === "Wooden Pickaxe");
-          message.member.addRole(role);
-      }
-      if(I.pick >= 1){
-          var role = message.guild.roles.find(role => role.name === "Stone Pickaxe");
-          message.member.addRole(role);
-      }
-      if(I.pick >= 2){
-          var role = message.guild.roles.find(role => role.name === "Iron Pickaxe");
-          message.member.addRole(role);
-      }
-      if(I.pick >= 3){
-          var role = message.guild.roles.find(role => role.name === "Diamond Pickaxe");
-          message.member.addRole(role);
-      }
+var addPickaxeRoles = (message, I) => {
+    if (message.channel.type !== 'dm' && (bot.guilds.get(message.guild.id).id).toString() === "550036987772403714") {
+        if (I.pick >= 0) {
+            var role = message.guild.roles.find(role => role.name === "Wooden Pickaxe");
+            message.member.addRole(role);
+        }
+        if (I.pick >= 1) {
+            var role = message.guild.roles.find(role => role.name === "Stone Pickaxe");
+            message.member.addRole(role);
+        }
+        if (I.pick >= 2) {
+            var role = message.guild.roles.find(role => role.name === "Iron Pickaxe");
+            message.member.addRole(role);
+        }
+        if (I.pick >= 3) {
+            var role = message.guild.roles.find(role => role.name === "Diamond Pickaxe");
+            message.member.addRole(role);
+        }
     }
 };
-var stone = "<:stone:550125781842395136>";
-var iron = "<:iron:550125781859172382>";
-var coal = "<:coal:550125781720891399>";
-var gold = "<:gold:550738153405677578>";
-var redstone = "<:redstone:550738154911432704>";
-var lapis = "<:lapis:550738153896542212>";
-var diamond = "<:diamond:550738152797765652>";
-var random = "<:random:550738153414328405>";
-var pickaxes = ["<:wooden_pickaxe:550733904051437570>", "<:stone_pickaxe:550733903896379445>", "<:iron_pickaxe:550049621477425176>", "<:diamond_pickaxe:550733904009625612>"];
-var air = "<:darkenedstone:550335949909786625>";
-var xps = ["<:xp_0:550744679550025728>", "<:xp_1:550744679327596545>", "<:xp_2:550744679495237643>", "<:xp_3:550744679680049167>", 
-        "<:xp_4:550744679776256050>", "<:xp_5:550744679830781967>", 
-        "<:xp_6:550744679910473753>", "<:xp_7:550744679562608642>", "<:xp_8:550744679835107340>"];
-var xp = "<:xp:550773197398736928>";
+var resources = {
```

Comment:
Instead of using a bunch of vars for these, I thought it would be better to just do this instead to make it needed and nice :}